skbio/alignment/_tabular_msa.py

```diff
@@ -685,13 +674,14 @@ def reindex(self, key=None, keys=None):
         array(['a', 'b'], dtype=object)
 
         """
-        if key is not None and keys is not None:
+        if minter is not None and keys is not None:
             raise ValueError(
-                "Cannot use both `key` and `keys` at the same time.")
+                "Cannot use both `minter` and `keys` at the same time.")
 
         keys_ = None
-        if key is not None:
-            keys_ = [resolve_key(seq, key) for seq in self._seqs]
+        if minter is not None:
```

Comment:
Yeah, reindexing without keys or a minter should remove the keys and the minter. Do you have a question about the comment you linked to? I previously followed up on that one.